internal/app/machined/pkg/controllers/network/nftables_chain_test.go

```diff
@@ -155,7 +155,7 @@ func (s *NfTablesChainSuite) TestICMPLimit() {
 	s.checkNftOutput(`table inet talos-test {
 	chain test1 {
 		type filter hook input priority security; policy accept;
-		meta l4proto icmp limit rate 5/second accept
+		meta l4proto icmp limit rate 5/second burst 5 packets accept
```

Comment:
I'm not sure we test in the CI actually (I don't think we have CLI there), so it might be my local CLI update. 

But we do set the burst, and previously `nft` didn't decode that properly.